code/modules/research/designs/nanotrasen.dm

```diff
@@ -20,6 +20,16 @@
 	locked = 1
 	req_lock_access = list(access_armory)
 
+/datum/design/mami
+	name = "Machine-Man Interface"
+	desc = "A synthetic brain interface intended to give silicon-based minds control of organic tissue. It's semi-organic, so store it in a cool, dry place."
+	id = "mami"
+	req_tech = list(Tc_PROGRAMMING = 4, Tc_BIOTECH = 4)
```

Comment:
doesn't this actually need ``Tc_NANOTRASEN = 1`` tech? Otherwise you can just print it with programming and biotech